functions/__tests__/functions.test.ts

```diff
@@ -0,0 +1,134 @@
+import * as functionsTestInit from 'firebase-functions-test'
+import mockedEnv from 'mocked-env'
+import { mocked } from 'ts-jest/utils'
+import { MeiliSearch } from 'meilisearch'
+
+jest.mock('meilisearch')
+
+const defaultEnvironment = {
+  LOCATION: 'us-central1',
+  PROJECT_ID: 'fake-project',
+  COLLECTION_PATH: 'example',
+  INPUT_FIELD_NAME: '',
+  MEILISEARCH_INDEX_NAME: 'example',
+  MEILISEARCH_HOST: 'http://127.0.0.1:7700',
+  MEILISEARCH_API_KEY: 'masterKey',
+}
+
+export const mockExport = (document, data) => {
+  const ref = require('../src/index').indexingWorker
+  const functionsTest = functionsTestInit()
+
+  const wrapped = functionsTest.wrap(ref)
+  return wrapped(document, data)
+}
+
+let restoreEnv
+const functionsTest = functionsTestInit()
```

Comment:
What is the purpose of this line? Can we not use `functionsTestInit()` when needed? 
Example: 

```js
// prev functionsTest.firestore...
const beforeSnapshot = functionsTestInit().firestore.makeDocumentSnapshot(
      {},
      'collection/doc'
    )
```

Reply:
It didn't work well did you try it? I try to find why but I can find it in the documentation

Reply:
I changed all the tests and it is working well. Did you add the `()` after `functionsTestInit` ?

Reply:
See new review

Reply:
after discussion it should be removed and added in `describe` 
```suggestion
```